<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(48, 47, 47);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Ctrl_L">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>310</y>
      <width>51</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string>Ctrl</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Shift_L">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>265</y>
      <width>81</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string>Shift</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Caps">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>220</y>
      <width>71</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string>Caps</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Tab">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>175</y>
      <width>61</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string>Tab</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Wave">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>130</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="One">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Two">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>130</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Three">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Four">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>130</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Seven">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>130</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Eight">
    <property name="geometry">
     <rect>
      <x>365</x>
      <y>130</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Nine">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>130</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Five">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>130</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Six">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>130</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Zero">
    <property name="geometry">
     <rect>
      <x>455</x>
      <y>130</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Bar">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>130</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Plus">
    <property name="geometry">
     <rect>
      <x>545</x>
      <y>130</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Backspace">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>130</y>
      <width>81</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string>Backspace</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Win_L">
    <property name="geometry">
     <rect>
      <x>56</x>
      <y>310</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string>Win</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Ait">
    <property name="geometry">
     <rect>
      <x>101</x>
      <y>310</y>
      <width>51</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string>Alt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Zz">
    <property name="geometry">
     <rect>
      <x>86</x>
      <y>265</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Xx">
    <property name="geometry">
     <rect>
      <x>131</x>
      <y>265</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Vv">
    <property name="geometry">
     <rect>
      <x>221</x>
      <y>265</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Cc">
    <property name="geometry">
     <rect>
      <x>176</x>
      <y>265</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Bb">
    <property name="geometry">
     <rect>
      <x>266</x>
      <y>265</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Mm">
    <property name="geometry">
     <rect>
      <x>356</x>
      <y>265</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Rest">
    <property name="geometry">
     <rect>
      <x>401</x>
      <y>265</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Spot">
    <property name="geometry">
     <rect>
      <x>446</x>
      <y>265</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Nn">
    <property name="geometry">
     <rect>
      <x>311</x>
      <y>265</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Slash">
    <property name="geometry">
     <rect>
      <x>491</x>
      <y>265</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Shift_R">
    <property name="geometry">
     <rect>
      <x>536</x>
      <y>265</y>
      <width>136</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string>Shift</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Ctrl_R">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>310</y>
      <width>51</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string>Ctrl</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Change">
    <property name="geometry">
     <rect>
      <x>479</x>
      <y>310</y>
      <width>51</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Menu">
    <property name="geometry">
     <rect>
      <x>575</x>
      <y>310</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string>Menu</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Win_R">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>310</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string>Win</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Space">
    <property name="geometry">
     <rect>
      <x>152</x>
      <y>310</y>
      <width>327</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string>Space</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Aa">
    <property name="geometry">
     <rect>
      <x>76</x>
      <y>220</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Ss">
    <property name="geometry">
     <rect>
      <x>121</x>
      <y>220</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Dd">
    <property name="geometry">
     <rect>
      <x>166</x>
      <y>220</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Ff">
    <property name="geometry">
     <rect>
      <x>211</x>
      <y>220</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Gg">
    <property name="geometry">
     <rect>
      <x>256</x>
      <y>220</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Hh">
    <property name="geometry">
     <rect>
      <x>301</x>
      <y>220</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Jj">
    <property name="geometry">
     <rect>
      <x>346</x>
      <y>220</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Kk">
    <property name="geometry">
     <rect>
      <x>391</x>
      <y>220</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Ll">
    <property name="geometry">
     <rect>
      <x>436</x>
      <y>220</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Semicolon">
    <property name="geometry">
     <rect>
      <x>481</x>
      <y>220</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Quotes">
    <property name="geometry">
     <rect>
      <x>526</x>
      <y>220</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Enter">
    <property name="geometry">
     <rect>
      <x>571</x>
      <y>220</y>
      <width>101</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string>Enter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Clear">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>70</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Copy">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>15</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string>Copy</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Qq">
    <property name="geometry">
     <rect>
      <x>66</x>
      <y>175</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Ww">
    <property name="geometry">
     <rect>
      <x>111</x>
      <y>175</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Ee">
    <property name="geometry">
     <rect>
      <x>156</x>
      <y>175</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Rr">
    <property name="geometry">
     <rect>
      <x>201</x>
      <y>175</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Tt">
    <property name="geometry">
     <rect>
      <x>246</x>
      <y>175</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Uu">
    <property name="geometry">
     <rect>
      <x>336</x>
      <y>175</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Yy">
    <property name="geometry">
     <rect>
      <x>291</x>
      <y>175</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Ii">
    <property name="geometry">
     <rect>
      <x>381</x>
      <y>175</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Pp">
    <property name="geometry">
     <rect>
      <x>471</x>
      <y>175</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Oo">
    <property name="geometry">
     <rect>
      <x>426</x>
      <y>175</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Parenthesis_Start">
    <property name="geometry">
     <rect>
      <x>516</x>
      <y>175</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Parenthesis_Finsh">
    <property name="geometry">
     <rect>
      <x>561</x>
      <y>175</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="ReverseSlash">
    <property name="geometry">
     <rect>
      <x>606</x>
      <y>175</y>
      <width>65</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Screen">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>601</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: gray;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
